modules/build/src/main/scala/scala/build/Build.scala

```diff
@@ -948,6 +950,8 @@ object Build {
 
     val project = Project(
       directory = inputs.workspace / Constants.workspaceDirName,
+      argsFilePath =
+        projectRootDir(inputs.workspace, inputs.projectName) / Constants.scalacArgumentsFileName,
```

Comment:
Where does it end up in `.scala-build` ?

Reply:
In `.scala-build/project-name-12345-HASH-12345/scalac.args.txt`, so it's kept together with the hashed contents of the compilation process, it should not get accessed by some other call to compile with different configuration

Reply:
Cool, thanks!